tests/comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1519,15 +1519,17 @@ d //comment
 \`
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 \`
-\${a // comment
+\${
+  a // comment
```

Comment:
I personally think this is better, but I can understand if this is considered a regression...